### PR TITLE
chore: added issue templates for the repo

### PR DESCRIPTION
**Pull request by anikethsaha**
Added issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. (from [cssnano-playground](https://cssnano.co/playground/))
+
+**Desktop (please complete the following information):**
+- CSSNANO Version [e.g. 22] 
+-  Plugin version (if it's known)
+<!-- also mention the version in which it was version (if any) -->
```
Comment (evilebottnawi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Will be great to recommend tool like `env` + `npm ls cssnano` to collect all information

Comment (anikethsaha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: any suggestion for tool which gives that without like without  adding it to the cli?


> `npm ls cssnano`

Ok cool

Comment (evilebottnawi) [reply .github/ISSUE_TEMPLATE/bug_report.md]: `envinfo` good tool

Comment (anikethsaha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: ok cool .. I will add `npx envinfo` in it and `npm ls cssnano` as well

Comment (evilebottnawi) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yep

Comment (anikethsaha) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: done

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. (from [cssnano-playground](https://cssnano.co/playground/))
+
+**Desktop (please complete the following information):**
+- CSSNANO Version [e.g. 22] 
+- Plugin version (if it's known)
+- Add the output of `npx envinfo`(recommended) or `npm ls cssnano` here
```
Comment (evilebottnawi) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: We need to have an one command and developers should put all information here, it is very bad to write `if it's known`

Comment (anikethsaha) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: yes, does makes sense.
removing that .

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. (from [cssnano-playground](https://cssnano.co/playground/))
+
+**Desktop (please complete the following information):**
+- CSSNANO Version [e.g. 22] 
+- Plugins/presets versions
+- Add the output of `npx envinfo`(recommended) or `npm ls cssnano` here
```
Comment (evilebottnawi) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: `npx envinfo && npm ls cssnano`

**Pull request merged**